test/specs/reference/tests/12.reference_values_edit.js

```diff
@@ -330,6 +330,7 @@ exports.execute = function (options) {
                     var jsonb=JSON.stringify(expectedValues[i].jsonb_col,undefined,2);
                     expect(values[1]).toBe(json);
```

Comment:
This is something that I should have mentioned in your previous PR. But when in a single `it` there are multiple `expect`s, add a message for each of them so you can understand why it failed. For example for this one it can be
```javascript
expect(values[1]).toBe(json, "mismatch in tuple with index=" + i + ", column=json_col");
```
Change other test cases accordingly.